src/templates/templates.json

```diff
@@ -43,6 +43,11 @@
                                             "type": "string",
                                             "default": null,
                                             "description": "If specified, adds a --docker-network parameter to the SAM CLI.  If not specified, no parameter will be sent."
+                                        },
+                                        "useContainer": {
+                                            "type": "boolean",
+                                            "default": false,
+                                            "description": "If specified, adds a --use-container parameter to the SAM CLI. If not specified, no parameter will be sent. If your functions depend on packages that have natively compiled dependencies use this flag to build your function inside an AWS Lambda-like Docker container."
```

Comment:
nit: `AWS Lambda-like Docker container` should be `SAM Docker container`

Reply:
Text is currently sourced from https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-cli-command-reference-sam-build.html

Reply:
I disagree, the text is weird and should not be copied